paper/paper.md

```diff
@@ -1,42 +1,45 @@
+title: 'gamut: A Geospatial R Package to Analyze Multisector Urban Teleconnections'
 tags:
-  - R
-  - Multisector Dynamics
-  - Water
-  - Energy
-  - Land
-  - Urban
-  - Geospatial
+- R
+- Multisector Dynamics
+- Water
+- Energy
+- Land
+- Urban
+- Geospatial
+date: "30 September 2021"
+output:
+  html_document: 
+    df_print: paged
 authors:
-  - name: Kristian D. Nelson
-    orcid: 0000-0002-6745-167X
-    affiliation: 1
-  - name: Sean W. Turner
-    orcid: 0000-0003-4400-9800
-    affiliation: 1
-  - name: Chris Vernon
-    orcid: 0000-0002-3406-6214
-    affiliation: 1
-  - name: Jennie Rice
-    orcid: 0000-0002-7833-9456
-    affiliation: 1
-affiliations:
- - name: Pacific Northwest National Laboratory
-   index: 1
-date: 26 April 2020
+- name: Kristian D. Nelson
+  orcid: 0000-0002-6745-167X
+  affiliation: 1
+- name: Sean W. Turner
+  orcid: 0000-0003-4400-9800
+  affiliation: 1
+- name: Chris Vernon
+  orcid: 0000-0002-3406-6214
+  affiliation: 1
+- name: Jennie Rice
+  orcid: 0000-0002-7833-9456
+  affiliation: 1
 bibliography: paper.bib
+affiliations:
+- name: Pacific Northwest National Laboratory
+  index: 1
 ---
 
 ### Summary
 
-Most cities in the United States withdraw surface water to meet public water supply needs. The lands on which this water is generated are often developed for human activities&mdash;such as agriculture, mining, and industry&mdash;that may compete for water resources or contaminate water supplies. Cities are thereby connected to other sectors through their water supply catchments. This connection is an example of an multisectoral urban teleconnection. The Geospatial Analytics for Multisectoral Urban Teleconnections (`gamut`) package provides national-scale information on these teleconnections by combining land use data with hydrological analysis to characterize urban source watershed human interactions across the conterminous United States (Figure 1).
+Most cities in the United States withdraw surface water to meet public water supply needs. The lands on which this water is generated are often developed for human activities&mdash;such as agriculture, mining, and industry&mdash;that may compete for water resources or contaminate water supplies. Cities are thereby connected to other sectors through their water supply catchments. This connection is an example of an multisectoral urban teleconnection, which is a "conceptual framework that explicitly links land changes to underlying urbanization dynamics" [@Seto:2012]. This term was brought about to bring greater understanding of the connections between urbanization and land use changes [@Seto:2012]. The Geospatial Analytics for Multisectoral Urban Teleconnections (`gamut`) package provides national-scale information on these urban teleconnections by combining land use data with hydrological analysis to characterize urban source watershed human interactions across the conterminous United States (Figure 1).
```

Comment:
We generally try to not directly quote other papers if possible.  

Instead of what you have here:

>  This connection is an example of an multisectoral urban teleconnection, which is a "conceptual framework that explicitly links land changes to underlying urbanization dynamics" [@Seto:2012]. This term was brought about to bring greater understanding of the connections between urbanization and land use changes [@Seto:2012]. 

Try something like:

> These connections are an example of a multisectoral urban teleconnection, or interdependency to a geographically disparate region from a source region where events in one (e.g., land use changes) often impact the other [@Seto:2012].